src/Compilers/CSharp/Portable/PublicAPI.Shipped.txt

```diff
@@ -85,6 +93,7 @@ Microsoft.CodeAnalysis.CSharp.Conversion.Equals(Microsoft.CodeAnalysis.CSharp.Co
 Microsoft.CodeAnalysis.CSharp.Conversion.Exists.get -> bool
 Microsoft.CodeAnalysis.CSharp.Conversion.IsAnonymousFunction.get -> bool
 Microsoft.CodeAnalysis.CSharp.Conversion.IsBoxing.get -> bool
+Microsoft.CodeAnalysis.CSharp.Conversion.IsCollectionLiteral.get -> bool
```

Comment:
@cston Just want to confirm we intend to keep this API name (as opposed to `IsCollectionExpression` or some such)

Reply:
This property was renamed in 17.8 to `IsCollectionExpression` (see https://github.com/dotnet/roslyn/pull/69114).

Reply:
For cases like this, do we just resolve when we ship the 17.8 as a remove/add?

Reply:
@cston What do you think of keeping `IsCollectionLiteral` as unshipped in 17.7?

Reply:
This probably needs to be renamed. Unless we also have IsTupleLiteral for the conversion as well.

Reply:
> What do you think of keeping `IsCollectionLiteral` as unshipped in 17.7?

Does that mean, leave `IsCollectionLiteral` in Public.Unshipped.txt for 17.7? What would that affect?

Reply:
You can also mark this as shipped but have a **REMOVE** line in the unshipped for 17.8. It makes it clearer that it would be a break in that case.

Reply:
Either seems fine to me.
Since the collection expressions feature is only in preview in 17.7, it seems fine to leave this API as unshipped. But we can also add it and remove it later...

Reply:
Let's leave it as unshipped.